### PR TITLE
4181 injection point pmr

### PR DESCRIPTION
**Pull request by benjamin-confino**

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

File: dev/com.ibm.ws.cdi.api_fat/publish/servers/cdi12InjectInjectionPointParamServer/jvm.options [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Set a system property.
+-Dcom.ibm.ws.webbeans.crossEjbModuleBindings=true
```
Comment (tevans78) [inline dev/com.ibm.ws.cdi.api_fat/publish/servers/cdi12InjectInjectionPointParamServer/jvm.options]: What??

Comment (Emily-Jiang) [reply dev/com.ibm.ws.cdi.api_fat/publish/servers/cdi12InjectInjectionPointParamServer/jvm.options]: this is for cdi 1.0. It must a c+p error

Comment (benjamin-confino) [reply dev/com.ibm.ws.cdi.api_fat/publish/servers/cdi12InjectInjectionPointParamServer/jvm.options]: It is a c+p error, I copied the entire server folder.

**Pull request merged**